client/src/components/SearchBar.js

```diff
@@ -3,19 +3,56 @@ import AdvancedSearch from "components/AdvancedSearch"
 import { SearchDescription } from "components/SearchFilters"
 import _isEqual from "lodash/isEqual"
 import PropTypes from "prop-types"
-import React, { useEffect, useRef, useState } from "react"
-import {
-  Button,
-  Form,
-  FormControl,
-  InputGroup,
-  Overlay,
-  Popover
-} from "react-bootstrap"
+import React, { useEffect, useRef, useState, PureComponent } from "react"
+import { Button, Form, FormControl, InputGroup } from "react-bootstrap"
 import { connect } from "react-redux"
 import { useHistory } from "react-router-dom"
 import SEARCH_ICON from "resources/search-alt.png"
 
+import { Popover, Position, PopoverInteractionKind } from "@blueprintjs/core"
```

Comment:
Please honour the correct ordering of imports, and add no blank lines between imports.